docs/plugins/harbor.md

```diff
@@ -0,0 +1,93 @@
+# harbor Plugin
+
+This plugin installs [harbor](https://goharbor.io/) in an existing Kubernetes cluster using the Helm chart.
+
+## Usage
+
+##### External storage
+
+- Postgresql: Set the `database.type` to `external` and fill the information in `database.external` section.
+- Redis: Set the `redis.type` to `external` and fill the information in `redis.external` section.
```

Comment:
Here I think it's a bit ambiguous: can I choose either Postgres or Redis, or must I have both?

Reply:
I will add a note for this